plugins/woocommerce/client/blocks/packages/components/radio-control/index.tsx

```diff
@@ -20,6 +20,7 @@ const RadioControl = ( {
 	options = [],
 	disabled = false,
 	highlightChecked = false,
+	descriptionStackingDirection = 'row',
```

Comment:
Curious as to why we're defaulting to `row` if we've introduced this prop to use `column`? `column` as a default kind of makes more sense to me but just curious as to the reason, does `row` more accurately reflect the behavior prior to this pr?

Reply:
Because this radio control is used by several other components. `row` is the default behaviour and we only want to apply `column` when it's used in the shipping methods block.

Reply:
It's the current behavior, I wanted to retain backwards compatibility.